docs/admin/namespaces/index.md

```diff
@@ -38,12 +38,12 @@ For this exercise, we will create two additional Kubernetes namespaces to hold o
 
 Let's imagine a scenario where an organization is using a shared Kubernetes cluster for development and production use cases.
 
-The development team would like to maintain a space in the cluster where they can get a view on the list of pods, services, and replication controllers
+The development team would like to maintain a space in the cluster where they can get a view on the list of Pods, Services, and Deployments
 they use to build and run their application.  In this space, Kubernetes resources come and go, and the restrictions on who can or cannot modify resources
 are relaxed to enable agile development.
 
 The operations team would like to maintain a space in the cluster where they can enforce strict procedures on who can or cannot manipulate the set of
-pods, services, and replication controllers that run the production site.
+pods, services, and Deployments that run the production site.
```

Comment:
Nit (can be fixed in followup): Deployments has different capitalization than pods and services.